src/js/apps/patients/view/view_app.js

```diff
@@ -61,10 +61,12 @@ export default App.extend({
   },
   _filtersById(viewId, currentClinician) {
     const clinician = currentClinician.id;
+    const roles = Radio.request('entities', 'roles:collection');
+    const role = roles.find({ short_name: 'CO' });
```

Comment:
did this work? seems like you still need `.id` on this line

Reply:
whoopsies. I thought it worked because it didn't blow up...but that's just because it wasn't passing the role filter at all...